commercetools/symfony-bundle/0.4/config/packages/security.yaml

```diff
@@ -0,0 +1,22 @@
+security:
+    providers:
+        ctp:
+            id: Commercetools\Symfony\CustomerBundle\Security\User\UserProvider
+    access_control:
+        - { path: /user/, roles: ROLE_USER }
+    encoders:
+        Symfony\Component\Security\Core\User\User: plaintext
+        Commercetools\Symfony\CustomerBundle\Security\User\User: plaintext
+    firewalls:
```

Comment:
Do we really need these security rules?

Reply:
Sorry for being short. 
I meant: The user will probably have a firewall setup already. But Im happy with you adding one new encoder.

Reply:
Well to be honest, I wasn't sure what the behavior of flex would be in case there is already a security.yaml file. ie if it was going to overwrite the file (?). So, I completely removed security.yaml now